packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -316,6 +316,9 @@ export class CallSubscriber {
         undefined,
         convertSdkLocalStreamToDeclarativeLocalStream(event.removed[0])
       );
+      if (event.removed[0].mediaStreamType === 'ScreenSharing') {
```

Comment:
Can you add a comment about this line and that we need to support local screen share streams

Reply:
will do!